hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/DeltaSync.java

```diff
@@ -221,7 +221,8 @@ private void refreshTimeline() throws IOException {
     } else {
       this.commitTimelineOpt = Option.empty();
       HoodieTableMetaClient.initTableType(new Configuration(jssc.hadoopConfiguration()), cfg.targetBasePath,
-          HoodieTableType.valueOf(cfg.tableType), cfg.targetTableName, "archived", cfg.payloadClassName, cfg.baseFileFormat);
+          HoodieTableType.valueOf(cfg.tableType), cfg.targetTableName, "archived", cfg.payloadClassName, cfg.baseFileFormat,
+          getHoodieClientConfig(this.schemaProvider).getIndexType().name());
```

Comment:
getHoodieClientConfig() does a bunch of things. Can we pass some object or use an existing member and avoid calling getHoodieClientConfig() again here?